test/integration/staking-rewards.ts

```diff
@@ -625,3 +625,19 @@ async function getUsersCurrentLpBalance() {
 
   return { totalDepositedLpTokens_d18, depositedLPTokenUser1_d18, depositedLPTokenUser2_d18 };
 }
+
+async function getStakingRewardsCount(): Promise<number> {
```

Comment:
Seems to me like this belong in the deploymentHelpers file, let's move it there?